modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -280,6 +364,56 @@ const Main = ({
 		itemSelectorDialog.open();
 	};
 
+	const configureErrorMessage = (message) => {
+		const enable = message ? true : false;
+
+		setErrorMessage(message);
```

Comment:
@alinedoleron, please declare the variables just before its use. 😉 
```
	const configureErrorMessage = (message) => {
		setErrorMessage(message);

		const enable = message ? true : false;

		setDisplayErrors(enable);
		setValid(!enable);
	};
```

Reply:
Sorry, I didn't get your point here.
Isn't the enable variable before the usage yet?